twint/run.py

```diff
@@ -192,14 +192,50 @@ async def profile(self):
 
     async def tweets(self):
         await self.Feed()
-        if self.config.Location:
-            logme.debug(__name__+':Twint:tweets:location')
-            self.count += await get.Multi(self.feed, self.config, self.conn)
-        else:
-            logme.debug(__name__+':Twint:tweets:notLocation')
-            for tweet in self.feed:
-                self.count += 1
-                await output.Tweets(tweet, self.config, self.conn)
+        search_tweet_list = []
+        for tweet in self.feed:
+            tweet_dict = {}
+            self.count += 1
+            try:
+                tweet_dict['data-item-id'] = tweet.find("div", {"class": "tweet-text"})['data-id']
+                t_url = tweet.find("span", {"class": "metadata"}).find("a")["href"]
+                tweet_dict['data-conversation-id'] = t_url.split('?')[0].split('/')[-1]
+                tweet_dict['username'] = tweet.find("div", {"class": "username"}).text.replace('\n', '').replace(' ',
+                                                                                                                 '')
+                tweet_dict['tweet'] = tweet.find("div", {"class": "tweet-text"}).find("div", {"class": "dir-ltr"}).text
+                date_str = tweet.find("td", {"class": "timestamp"}).find("a").text
+
+                if len(date_str) <= 3 and (date_str[-1] == "m" or date_str[-1] == "h"):  # 25m 1h
```

Comment:
@yunusemrecatalcam. I think you also need to account for seconds here. Otherwise it will throw an exception. 
I think this would work:
`if len(date_str) <= 3 and (date_str[-1] == "m" or date_str[-1] == "h" or date_str[-1] == "s"):  # 25m 1h`